pkg/amqp-ext/Flags.php

```diff
@@ -2,11 +2,11 @@
 
 namespace Enqueue\AmqpExt;
 
-use Interop\Amqp\AmqpConsumer;
```

Comment:
That would be enought to remove this single line `use Interop\Amqp\AmqpConsumer;`. It must work.

Reply:
Since AmqpConsumer from Enqueue implements InteropAmqpConsumer, it should work indeed! I'm gonna give it a quick test and keep you posted.

Reply:
Works! Confirmed by testing on 5.6.31. Should work on other versions as well. let's go for it :)

Reply:
@chdeliens great! Could you please update the PR?